bjoern/request.c

```diff
@@ -31,7 +37,9 @@ void Request_reset(Request* request)
 {
   memset(&request->state, 0, sizeof(Request) - (size_t)&((Request*)NULL)->state);
   request->state.response_length_unknown = true;
-  request->parser.body = (string){NULL, 0};
+  request->parser.last_was_data = true;
```

Comment:
Can we rename this to something along the lines of `last_call_was_header_value`?

Reply:
good point, renamed